flytekit/core/base_task.py

```diff
@@ -791,6 +792,15 @@ def execute(self, **kwargs) -> Any:
         """
         pass
 
+    def remote(self, version=None, **kwargs) -> FlyteFuture:
+        """
+        This method will be invoked to execute the task remotely. This will return a FlyteFuture object that can be
+        used to track the progress of the task execution.
+
+        This method should be executed after specifying the remote configuration via `flytekit.remote.init_remote()`.
```

Comment:
Let's write docs for all arguments and outputs.
Especially because we are mirroring Flytekit.remote, but folks will just start using this interface.

Reply:
Will it raise an exception if the remote is not initialized?